src/_colors.scss

```diff
@@ -8,6 +8,7 @@ $ui-orange: hsla(38, 100, 55, 1); // #FFAB19 Control Primary
 $ui-orange-high-contrast: hsla(30, 100, 55, 1); // #FFAB19 Control Primary
 $ui-orange-10percent: hsla(35, 90, 55, .1);
 $ui-orange-25percent: hsla(35, 90, 55, .25);
+$ui-orange-90percent: hsla(38, 100, 55, .9);
```

Comment:
I was expecting there to be consistency between our naming of something as ui-color-YYpercent and the saturation for those colors. Seems like that isn't the case though, right? i.e. the 100 here is not consistent with the 90 for the ui-orange-10percent and ui-orange-25percent.  As long as that inconsistency is expected, LGTM

Reply:
I think the 90 percent thing is confusing here because there are really three kinds of orange that we are treating as if they're two; my 90 percent uses the same base orange as $ui-orange, whereas 10 percent and 25 percent don't. Since 90 percent is so much closer to 100 percent than to 10 and 25 percent, this feels like the most consistent choice.

The "percent" in the other color names refers to brightness/alpha (which is the same thing on a white background).